cli/pkg/kctrl/cmd/kctrl.go

```diff
@@ -76,7 +76,8 @@ func NewKctrlCmd(o *KctrlOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Comm
 		cmdcore.RestOfCommandsHelpGroup,
 	}))
 
-	pkgOpts := cmdcore.PackageCommandTreeOpts{BinaryName: "kctrl", PositionalArgs: false, Color: true, JSON: true}
+	pkgOpts := cmdcore.PackageCommandTreeOpts{BinaryName: "kctrl", PositionalArgs: false, Color: true, JSON: true,
```

Comment:
nit: I think we can remove `PositionalArgs` and `AllowSharedNamespaces` as the default value should be false anyway.

Reply:
Not related to the current changes, but yeah that makes sense, I will make the change